add_glyphs.py

```diff
@@ -228,7 +228,10 @@ def is_fitzpatrick(cp):
 
   rev_seq = list(seq)
   rev_seq.reverse()
-  for i in xrange(1, len(rev_seq)):
+  starts_with_fp = is_fitzpatrick(rev_seq[0])
+  for i in range(1, len(rev_seq)):
+    if i == 2 and starts_with_fp:
+      continue
```

Comment:
this is a bit convoluted (it took me a bit to understand what was going on here). I suggest you do this instead:

```python
for i in range(len(rev_seq)-1, 0, -1):
     if is_fitzpatrick(rev_seq[i-1]):
         rev_seq[i-1], rev_seq[i] = rev_seq[i], rev_seq[i-1]
```

by looping over the rev_seq list from the end to the beginning, we don't end up swapping a "fitzpatrick" again and again all the way to the end of the loop

Reply:
I wasn't very happy with that code, your suggestion is a lot more elegant. Thanks!